cypress/integration/Train/TagAndFrog.spec.js

```diff
@@ -8,72 +8,92 @@ import * as modelPage from '../../support/components/ModelPage'
 import * as train from '../../support/Train'
 import * as editDialogModal from '../../support/components/EditDialogModal'
 import * as common from '../../support/Common'
+import * as helpers from '../../support/Helpers'
+
+describe('Tag And Frog - Train', () => {
+  // TODO: Need to add another test case or expand this one so that tagging something
+  //       that was NOT tagged in another instance causes the UI to complain.
```

Comment:
Hey Brian, this TODO comment is a good candidate for enhancing this test case next time you are looking for work to do.